src/plugins/experimental-discord/config.js

```diff
@@ -22,12 +22,22 @@ export type DiscordConfig = {|
   // You can get a custom emoji ID by right clicking the custom emoji and
   // copying it's URL, with the ID being the image file name.
   +reactionWeights: EmojiWeightMap,
+  // An object mapping a role to a weight, as in:
+  // {
+  //   "default": 0,
+  //   "core:626763367893303303": 2,
+  //   "contributor:456763457893303303": 1,
+  // }
+  // Note that roles have a snowflake identifier.
+  // default is used to set weights for members who don't have a specified role
+  +roleWeights: RoleWeightMap,
```

Comment:
Suggest having the default be an explicit property, rather than an entry in the map.

```js
type RoleWeightConfig = {|
  +defaultWeight: number,
  +roleWeights: $ReadOnlyMap<DiscordSnowflake, number>
|}
```

This is a lot clearer and easier to reason about than having the string key "default" play a special role.